node/subsystem-types/src/messages.rs

```diff
@@ -318,7 +318,36 @@ pub enum DisputeDistributionMessage {
 	SendDispute(DisputeMessage),
 }
 
-/// Messages received by the network bridge subsystem.
+/// Messages received from other subsystems.
+///
+/// A dummy.
+#[derive(Debug)]
+pub enum NetworkBridgeInMessage {
```

Comment:
Naming is hard, I wanted to keep it consistent with the `NetworkBridgeIn`, but that'd also be subject to change as needed. It should also be specific to what is received, since it the source could be either the external network or the other subsystems.